### ISSUE TITLE
Playlist Organization

### ISSUE THREAD

**Issue by ManoelNetoo** [fast4x/RiMusic#4791]
### Description

option to organize the songs that are within the created playlists.

### Suggested Solution

It is true that there is already a function in the app to organize the songs in the playlists created, however, it is only possible to organize the songs in alphabetical order (from A - Z or Z - A). It would be interesting to have the option to organize the order of the songs in the playlists in the way the user wants, that way, the user could organize their songs in the order they prefer, and listen to the songs they want in the order they defined without having to open the app to change the song or without having to skip songs until they get to the one they want to listen to. If you allow me to use an example app, Spotify has this function to organize the songs within the playlists created by the user, you can base it on it to implement in the app.

**Comment by fast4x** [comment]
Hi if you click on order type, can set others...
![Screenshot_2024-12-04-20-28-21-692_it.fast4x.rimusic-edit.jpg](https://github.com/user-attachments/assets/8cffcc85-d6a0-4a3e-ad7b-b58520efcfac)

![Screenshot_2024-12-04-20-28-12-768_it.fast4x.rimusic.jpg](https://github.com/user-attachments/assets/795e87a6-50d6-42a8-a013-323509a83504)

**Comment by ManoelNetoo** [comment]
Yes, I know there are other ways to organize, but all of these ways are
already predefined, whether in alphabetical order, by song duration,
artist, etc. It is not you who organizes, you cannot move the songs and
organize the order of the songs the way you want. My opinion is that in
addition to these options that are already in the app, there could be a
"custom" option for example, where in this option I can organize the songs
freely the way I want, so I can move the songs in the playlists and put
them in the order I want, without having a predefined organization. The app
is perfect, by the way, this is just one possibility. Congratulations to
everyone who developed it.

Em qua., 4 de dez. de 2024, 16:29, Rino Russo ***@***.***>
escreveu:

**Comment by aneesh1122** [comment]
https://github.com/user-attachments/assets/d53c1e56-4669-4059-8ee4-23ffe5a8c65a

**Comment by SergyManuto** [comment]
Is not working for me, when I try to move from the bottom to the top the song is bugged and did not move

**Comment by knighthat** [comment]
please make sure you're in "position" with "ascending" order (arrow up) to make sure changes are saved correctly

**Comment by SergyManuto** [comment]
> please make sure you're in "position" with "ascending" order (arrow up) to make sure changes are saved correctly

![image](https://github.com/user-attachments/assets/c18dc77c-5cfd-49ee-8430-4c70a2f805d5)
like this?

**Comment by knighthat** [comment]
Yes

**Comment by SergyManuto** [comment]
> Yes

I did it, in the video u can see it, but is bugged for me and I can't reorder the songs

**Comment by fast4x** [comment]
Will be improved
